Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
-ruby "2.3.4"
+ruby "2.5.1"
```

Comment:
@sirleech This fails to build in CCI. Apparently the Ruby version being run is still 2.3.4. Was this upgraded later?